src/utils.js

```diff
@@ -523,23 +553,25 @@ export function is_declaration(node) {
 
 /**
  * @param {ts.Node} node
+ * @param {boolean} [include_declarations]
  * @returns {node is ts.Identifier}
  */
-export function is_reference(node) {
+export function is_reference(node, include_declarations = false) {
 	if (!ts.isIdentifier(node)) return false;
 
 	if (node.parent) {
 		if (is_declaration(node.parent)) {
 			if (ts.isVariableStatement(node.parent)) {
-				return node === node.parent.declarationList.declarations[0].name;
+				return false;
```

Comment:
I changed this because the previous statement didn't make sense - the node could never be equal to the right hand side, because the node itself can only ever be `VariableDeclarationList`